src/rebuild/scan.c

```diff
@@ -471,7 +555,7 @@ rebuild_scanner(void *data)
 	struct umem_attr		uma;
 	int				rc;
 
-	if (!is_current_tgt_up(rpt)) {
+	if (is_current_tgt_unavail(rpt)) {
```

Comment:
I think we should in the scan phase.
Because the UP target could potentially be the leader for a given object when calling
the reintegration API as part of placement..

Reply:
we will choose an reintegrating target as the object leader?  hmm, ok, let's check later here.